sklearn/metrics/classification.py

```diff
@@ -1233,6 +1243,15 @@ def precision_recall_fscore_support(y_true, y_pred, beta=1.0, labels=None,
      array([0., 0., 1.]), array([0. , 0. , 0.8]),
      array([2, 2, 2]))
 
+    Notes
+    -----
+    When ``true_positive + false positive == 0``, precision will be set to 0
```

Comment:
I think you can avoid this repetition:

When true positive + false positive == 0, precision is undefined; when true positive + false negative == 0, recall is undefined. In such cases, the metric will be set to 0, as will f-score, and an UndefinedMetricWarning will be raised.